packages/web-components/src/components/ic-button/ic-button.tsx

```diff
@@ -177,6 +173,11 @@ export class Button {
    */
   @Prop() loading?: boolean = false;
 
+  /**
+   * If `true`, the breadcrumb will display as black in the light theme, and white in the dark theme.
```

Comment:
```suggestion
   * If `true`, the button will display as black in the light theme, and white in the dark theme.
```
might also want to reword as actually it's the opposite for secondary & tertiary buttons